lib/streamlit/runtime/app_session.py

```diff
@@ -502,6 +505,7 @@ def _handle_scriptrunner_event_on_event_loop(
         client_state: ClientState | None = None,
         page_script_hash: str | None = None,
         fragment_ids_this_run: set[str] | None = None,
+        pages: dict[PageHash, PageInfo] = {},
```

Comment:
Same comment about the default value used here

Reply:
This one seems a bit weird to remove. I can do that here, but it's just another handler tied to the event handler. Will leave for your thoughts.

Reply:
The correct way of doing this would be to have the default value be `None` and initialize it to an empty dict in the function if necessary.

Reply:
Ah. I gotcha here. I'll take a look!